### ISSUE TITLE
Making Multiple Instances of the Wrapper

### ISSUE THREAD

**Issue by parsasaeedi** [JMPerez/spotify-web-api-js#216]
I'm creating an app where I need 2 instances of the wrapper, one for each user/access-token. But when I make 2 instances, it seems like they are both the same instance.

```javascript
let SpotifyWebApi = require('spotify-web-api-js');
let spotifyApi1 = new SpotifyWebApi();
let spotifyApi2 = new SpotifyWebApi();

console.log(spotifyApi1.getAccessToken());
console.log(spotifyApi2.getAccessToken());

spotifyApi1.setAccessToken("Hello");
spotifyApi2.setAccessToken("Bye");

console.log(spotifyApi1.getAccessToken());
console.log(spotifyApi2.getAccessToken());
```

prints:

```
null
null
Bye
Bye
```

Where I expected it to print:

```
null
null
Hello
Bye
```

The access token in both instances is set to the latest set statement. 

Same thing even if I write:
```javascript
let SpotifyWebApi1 = require('spotify-web-api-js');
let SpotifyWebApi2 = require('spotify-web-api-js');
let spotifyApi1 = new SpotifyWebApi1();
let spotifyApi2 = new SpotifyWebApi2();
```

I don't know if this is a problem with the wrapper, or my lacking JavaScript skills. But I can't find a generic JavaScript answer on StackOverflow.

**Additional context**
I'm making a react app and I installed the wrapper package using npm.
[Here is my project](https://github.com/parsasaeedi/interspot). I instantiate the wrappers in /src/Interspot.js and use them in /src/useSpotifyAPI.js

**Comment by parsasaeedi** [comment]
https://stackoverflow.com/questions/73430871/making-multiple-instances-of-spotify-web-api-js

**Comment by JMPerez** [comment]
@parsasaeedi Yes, this looks definitely like a problem as your use case should be supported. This is a bug that should be addressed.
